solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -378,6 +382,11 @@ function logs_generation(){
     popd
 }
 
+function cleanup(){
+    echo -e "\n-----------[ Clean up untagged/unused images and stopped containers... ]--------------------"
+    docker system prune -a -f
```

Comment:
@nikhilpatil2995 , Is this enough for disk cleanup?

Reply:
If you see the tested job link which I mentioned in the ticket, with the help of this we almost clear more than 3Gb of disk space from VM